theseus_gui/src/components/ui/ExportModal.vue

```diff
@@ -123,11 +152,34 @@ const exportPack = async () => {
           </Button>
         </div>
       </div>
+      <div class="adjacent-input">
+        <div class="labeled_input">
+          <p>Description</p>
+
+          <div class="textarea-wrapper">
+            <textarea
+              v-model="exportDescription"
+              placeholder="My super-special modpack is the best of them all!"
```

Comment:
placeholders shouldn't be used this way. I would put something like "Enter modpack description..."